app/models/bar.py

```diff
@@ -85,7 +86,8 @@ def _update_linked_objects(self, revert: bool):
             daily_data.items_bought_count += factor * 1
             daily_data.total_spent -= factor * self.balance_change
             if self.item.is_alcohol:
-                daily_data.alcohol_bought_count += factor * 1
+                degree = 5 if self.item.alcohol_degree is None else self.item.alcohol_degree
```

Comment:
Pas fan de ça codé en dur, je préférerais passer `alcohol_degree` en NOT NULL et passer à 5 les items existants via la migration